components/debug/debugging.hpp

```diff
@@ -82,7 +82,8 @@ namespace Debug
             mColors[Error] = Red;
             mColors[Warning] = Yellow;
             mColors[Info] = Reset;
-            mColors[Verbose] = DarkGray;
+            mColors[Verbose] = Reset;
```

Comment:
Are you sure the color should be changed for Verbose level?

Reply:
Why it should not?

Reply:
Why should it?

It's been dark gray forever -- well, since the time you implemented the new logging system -- and that has been just fine for all the messages it was used for. At least I myself got used to it.

Reply:
Can you suggest an another color for Debug level then?

Reply:
IMO dark gray for both levels would have been just fine.

Reply:
Done.